### PR TITLE
Add information for Scaleway users

### PR DESCRIPTION
**Pull request by kirill-zak**
Add information section into `README.md` for Scaleway VPS users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,10 @@ To check the status of your IPsec VPN server, you can pass `ipsec status` to you
 docker exec -it l2tp-ipsec-vpn-server ipsec status
 ```
 
+### Scaleway user information
```
Comment (fcojean) [inline README.md:107]: Hi,
Scaleway seems to be a brand, so can you add the copyright symbol after Scaleway string please.
Example : Scaleway ©.

File: README.md [hunk 2]
```diff
@@ -104,6 +104,10 @@ To check the status of your IPsec VPN server, you can pass `ipsec status` to you
...
+
+Scaleway use own modified kernel version `4.4` by default. This kernel isn't compatible with IPsec. If you want to use IPsec VPN server on Scaleway VPS you should switch version of kernel (version `4.8` or `higher`).
```
Comment (fcojean) [inline README.md]: Can you add a link to Scaleway website on the Scaleway string please. So there is no misunderstanding on what is Scaleway.

Comment (fcojean) [reply README.md]: @kirill-zak : Are you a Scaleway employee or someone related to the Scaleway company ?

**Review by fcojean — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kirill-zak) [reply README.md:107]: Ok.
* I added copyrate mark
* I added link to Scaleway home page

**Review by fcojean — APPROVED** [review]

**Pull request merged**